src/multi-level-checkbox-editor/node-label.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+interface LabelDetails {
+    text: string;
+    info: string;
+}
+
+interface LabelInformation {
+    labels: Array<LabelDetails>;
+    note: string;
+}
+
+interface Props {
+    labelInformation?: LabelInformation;
+    label?: string;
+}
+
+const NodeLabel: React.FunctionComponent<Props> = (props) => (
+    <div className={'node-label'}>
+        {props.labelInformation &&
+            props.labelInformation.labels.map((label, index) => {
+                if (index > 1) {
```

Comment:
Isn't it easier to just pick the first and map it?

Reply:
done with slice